stdgo/reflect/Reflect.hx

```diff
@@ -587,16 +766,23 @@ function typeOfClass(cl:Class<Dynamic>, v:Dynamic):GoType {
 	}
 }
 
-var rttiCClassMap:Map<String, GoType> = ["String" => GT_string];
+var nonRttiClassMap:Map<String, Type> = [
+	"eval.Vector" => GT_array(GT_invalid, 0),
```

Comment:
Unfortunately the string mapping of Vector is target specific, so either many more target specific Vectors need to be added or GoArray's underlying type can be a Class that has Vector as a field, up to you.

Reply:
Unless we can find a more efficient way to pass in the Go type information, I'm afraid a huge number of target specific types will be needed here.

Reply:
Good point, for ``eval.Vector`` can at least be updated to use a cross target class in the near future. Target specific types hopefully can be handled by another means.